src/components/Modals/Nfts/components/NftCollection.tsx

```diff
@@ -32,16 +33,12 @@ export const NftCollection: React.FC<NftCollectionProps> = ({ zapperCollection }
         ))}
       </Flex>
     )
-  }, [collection?.socialLinks])
-
-  if (!collection) return null
-
-  const { description, name: collectionName } = collection
+  }, [socialLinks])
 
   return (
     <Flex gap={4} flexDir='column' px={8} py={6}>
       <Text fontWeight='medium'>{translate('nft.aboutCollection', { collectionName })}</Text>
-      <ParsedHtml color='gray.500' innerHtml={markdownLinkToHTML(description)} />
+      <ParsedHtml color='gray.500' innerHtml={markdownLinkToHTML(description || '')} />
```

Comment:
```suggestion
      <ParsedHtml color='gray.500' innerHtml={markdownLinkToHTML(description ?? '')} />
```

Reply:
ditto empty string needs a `||` check
<img width="109" alt="image" src="https://github.com/shapeshift/web/assets/17035424/272a715c-fc3f-42c5-b05f-687821d288d0">

Reply:
as mentioned in the other pr, i know the nullish coalesce doesn't catch the empty string, but only really useful to use `||` if we are defaulting something to other than `''` in that case.

Reply:
Will tackle as part of the last PR in the stack 🙏🏽